RELEASE.md

```diff
@@ -1,3 +1,111 @@
+# Release 1.1.0
+
+## Major Features and Improvements
+* Added Java API support for Windows.
+* Added `tf.spectral` module. Moved existing FFT ops to `tf.spectral`.
+* Added 1D, 2D and 3D Fourier transform ops for real signals to `tf.spectral`.
+* Added a `tf.bincount` function.
+* Added Keras 2 API to contrib.
+* Added a new lightweight queue-like object - `RecordInput`.
+* Added `tf.contrib.image.compose_transforms` function.
+* Bring `tf.estimator.*` into the API. Non-deprecated functionality from `tf.contrib.learn.Estimator` is moved to `tf.estimator.Estimator` with cosmetic changes.
+* Docker images: TF images on gcr.io and Docker Hub are upgraded to ubuntu:16.04.
+* Added the following features to TensorFlow Debugger (tfdbg):
+  * Ability to inspect Python source file against TF ops and tensors (command `print_source` / `ps`)
+  * New navigation bar in Curses-based UI
+  * NodeStepper (command `invoke_stepper`) now uses intermediate tensor dumps. It also uses `TensorHandles` as direct feeds during successive `cont` calls for improved performance and reduced memory consumption.
+
+## Deprecations
+
+* TensorFlow will stop supporting CUDA on MacOS due to instability of the
+  configuration, and inavailability of up-to-date high performance systems with
+  such support. 1.1.0 will be the last official release we will have for tensorflow-gpu on MacOS.
```

Comment:
Let's expand on this, starting with the last sentence:
We will discontinue building binaries, and support for compilation from sources will be from the community.
We will still accept community contributions to fix issues for this configuration.

@martinwicke PTAL.